akka-actor/src/main/scala/akka/util/ByteString.scala

```diff
@@ -528,7 +528,7 @@ object CompactByteString {
     else {
       val ar = new Array[Byte](bytes.size)
       bytes.copyToArray(ar)
-      CompactByteString(ar)
+      ByteString.ByteString1C(ar)
```

Comment:
I wonder if we can do even better by

```
else 
  ByteString.ByteString1C(bytes.toArray)
```

The varargs `bytes` is a `scala.collection.mutable.WrappedArray$ofByte` and the `toArray` is therefore a cast of the internal array. cloning is done by `ByteString1C`